lib/pure/net.nim

```diff
@@ -611,6 +647,23 @@ when defineSsl:
     if SSLSetFd(socket.sslHandle, socket.fd) != 1:
       raiseSSLError()
 
+  proc checkCertName*(socket: Socket, hostname: string) =
```

Comment:
Is this something that will be useful outside of this module?

Reply:
Good point, I'll hide it.